### PR TITLE
Add MTU to get_interfaces on nxos and iosxr

### PR DESCRIPTION
**Pull request by targuan**
Add MTU to get_interfaces for iosxr and nxos drivers. This is based on the mocked datas. Further testing may be required. 
Add MTU to the results of junos driver get_interfaces tests.

### CODE REVIEW

File: napalm/nxos/nxos.py [hunk 1]
```diff
@@ -400,6 +400,12 @@ def get_interfaces(self):
                 is_up = interface_details.get('admin_state', '') == 'up'
             else:
                 is_up = interface_details.get('state', '') == 'up'
+            if 'eth_mtu' in interface_details:
+                mtu = int(interface_details['eth_mtu'])
+            elif 'svi_mtu' in interface_details:
```
Comment (mirceaulinic) [inline napalm/nxos/nxos.py]: That is a bit awkward (but I understand why, I know the NX-API slickness). Would you please add a comment around that with more details, and - if applicable - what versions have one but not the other etc.

**Comment by targuan** [comment]
Hi,

I added some comments on the section that wasn't clear enough. Should be ok now.

**Comment by dbarrosop** [comment]
@targuan there seems to be a conflic, would you mind checking? thanks!

**Comment by dbarrosop** [comment]
I think your branch is broken:

* Commits 143
* Files changed 134 

I'd suggest creating a new branch and putting your code there.

**Comment by targuan** [comment]
Actually, just the pr is broken. Develop has been merge in the mean time and I did a merge instead of a rebase I guess. My git knowledge is not that Good... 

I'll close this pr and open a more clear one.

**Pull request closed without merging**